share/governance.md

```diff
@@ -0,0 +1,95 @@
+* providing moral support (a ‘thank you’ goes a long way)
+* writing documentation and tutorials
+* filing bug reports and feature requests
+* participating on the discussion board
+
+
+Users who continue to engage with the project and its community will often find themselves becoming more and more involved. Such users may then go on to become contributors, as described below.
+
+*How to become one*: Use a Vox Pupuli project
+
+
+Collaborators
+-------------
+
+
+Collaborators are community members who submit patches to the project. These patches may be a one-time occurrence or occur over time. Expectations are that contributors will submit patches that are small at first and will only grow larger once the contributor has built confidence in the quality of their patches.
+
+*How to become one*: Submit a pull request to a Vox Pupuli project
+
+
+Contributors
+------------
+
+
+Contributors are collaborators who have shown wide dedication to the Vox Pupuli project in general or deep dedication to one project in particular, and the ability to work well with collaborators and users. The contributors have responsibilities beyond the collaborators. In particular, contributors formally decide on whether a pull request is merged or a release is cut. Contributors are recognized and valued for their contributions, in whatever form those contributions take. Tasks such as communication, documentation, art, packaging, and testing are all valued contributions. A contributor will use lazy consensus to decide on whether to merge a pull request from a collaborator. If the discussion is no longer moving towards a consensus, the PMC must vote via lazy consensus on whether the patch should be applied.
+
+*How to become one*: Be a collaborator and be nominated to the PMC as a contributor. Nominations should be sent to <email address for nominations>. You may nominate yourself. It is also common to give contributor status to an individual who donates code to the project by migrating a repository to the github namespace.
+
+
+Project Management Committee (PMC)
+----------------------------------
+
+
+The project management committee has responsibilities beyond contributors that include participating in strategic planning, release planning, and approving changes to the governance model. They also have the duty of upholding the community code of conduct and ensuring its values. The PMC is also responsible for updating the community code of conduct. It also makes decisions when community consensus cannot be reached. 
+
+The PMC has final say over who can become a committer and will use lazy consensus for approval. Discussion over committer nominations will be done in private.
+
+Membership of the PMC is by election. Condorcet(https://en.wikipedia.org/wiki/Condorcet_method) voting is held once a year. A non-running election official is appointed by the PMC and runs the election. There are 5 members of the PMC. Terms are 1 year.
```

Comment:
Oh man I had a whole thing written then github lost it :(:( :(

For the exact period we won't know until we run one. I figure running one happens shortly after this document lands, then every year after that.

For election cycle length, that's up to the election official (again a place where it is easier on everyone to just let the election official handle it than write down an arbitrary uniformed number.)

For term lengths and term limits, I don't know, I considered it. I changed it to 1 year terms because the template had no term length, it was a lifetime appointment, and I figured we could do better than that. We're only 2ish years into voxpupuli, so setting 2 year terms seemed high. I don't know that we have enough people interested in governance to add term limits, it would be pretty terrible if we ran out of leaders.

The complete responsibilities of the election official? Who knows. Run the election! Again this is a thing where I don't expect to be complete so why even try.

As for what qualifications they have.... I don't know. I think non-running is a hard requirement, can we leave the other requirements up to the PMC, who have to select the person in the first place? That being said, we have to pick a first one, again this is a bootstrapping problem. There currently exists a precursor to the PMC, the project-maintainers, who I expect will pick the first election official.